### ISSUE TITLE
Just iterate 2 times

### ISSUE THREAD

**Issue by a1361098690** [brendanoconnor913/SFFS#1]
I distinguish the training set from the test set based on your code requirements and the corresponding tags. Why did I stop iterating twice, I have a total of 96 features.
![1](https://user-images.githubusercontent.com/45624213/59345097-fbe52580-8d41-11e9-85e7-15ad5965af72.jpg)
![2](https://user-images.githubusercontent.com/45624213/59345105-01427000-8d42-11e9-810a-26d30bd519ef.png)
